uint64 id
sensor_msgs/Range range
float64 std_dev
